### PR TITLE
bugfix/#511-footer

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**
Fixes for homepage, footer and header

### CODE REVIEW

File: src/components/Footer/Footer.module.scss [hunk 1]
```diff
@@ -5,7 +5,8 @@
   @include relative;
   padding-top: 140px;
   height: 720px;
-  background-color: $color-black;
+  max-width: 1440px;
```
Comment (ArekRado) [inline src/components/Footer/Footer.module.scss]: you can use container mixin

File: src/components/Header/Header.module.scss [hunk 2]
```diff
@@ -89,9 +90,11 @@
 
 .container {
   @include px(4);
+  max-width: 1440px;
```
Comment (ArekRado) [inline src/components/Header/Header.module.scss]: container mixin

File: src/components/Nav/Nav.js [hunk 3]
```diff
@@ -1,26 +1,33 @@
 import React from 'react'
 import cn from 'classnames'
-import pure from 'recompose/pure'
+import { pure, compose, withHandlers } from 'recompose'
```
Comment (ArekRado) [inline src/components/Nav/Nav.js]: right now we need to take care about imports. import each module from another import

```js
import pure from 'recompose/pure'
```

File: src/components/Header/Header.module.scss (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/components/Header/Header.module.scss]: I don't want flex in there

**Pull request merged**